.github/workflows/vm-kernel.yaml

```
@@ -42,6 +42,7 @@ jobs:
         id: build-linux-kernel
         uses: docker/build-push-action@v3
         with:
+          build-args: KERNEL_VERSION=6.1.63 # Temporary: Will be overridden by #662.
```

@bayandin IIUC this would kind of conflict with #662 - just want to make sure you're aware. Happy to handle it in this PR or that one, whichever one merges second.

---

After rebasing on main you can use 

```suggestion
          build-args: KERNEL_VERSION=${{ steps.get-kernel-version.outputs.VM_KERNEL_VERSION }}
```

---

Done, thanks :) c544b951c2d707a78a9fecb6a116479d3aad84d1